### PR TITLE
Bulk inserts for static assets association to learning resources

### LINKED ISSUE #697: Importing is slow (at least in local storage)
Using Overwrite for local storage triggers a signal for each M2M change

This blocks further development

### PR DESCRIPTION
**Pull request by giocalitri**
Closes #697

Depends on #704

### CODE REVIEW

File: importer/api/__init__.py [hunk 1]
```diff
@@ -193,11 +199,22 @@ def import_children(course, element, parent, parent_dpath):
                                 course__id=resource.course_id,
                                 asset=course_asset_basepath(course, path),
                             )
-                            resource.static_assets.add(asset)
+                            static_assets_to_save.add((resource, asset))
                         except StaticAsset.DoesNotExist:
                             continue
                 except AttributeError:
                     continue  # not a string
+    # bulk insert of static assets
```
Comment (noisecapella) [inline importer/api/__init__.py]: Can you update the comment to explain about the m2m_changed signal we're trying to avoid?

Comment (giocalitri) [inline importer/api/__init__.py]: will do

**Comment by giocalitri** [comment]
done

**Comment by noisecapella** [comment]
:+1: after tests pass

**Pull request merged**